packages/commonwealth/client/scripts/helpers/snapshot_utils.ts

```diff
@@ -235,6 +254,17 @@ export async function getSpace(space: string): Promise<SnapshotSpace> {
   return spaceObj.data.space;
 }
 
+export async function getProposal(id: string): Promise<{ title: string, space: string }> {
+  await getApolloClient();
```

Comment:
dose it work fine? `getApolloClient()` returns `apolloClient` instance so shouldn't it be like 
`const client = await getApolloClient();`

Reply:
The logic is bad but `getApolloClient()` is really an init call for the global variable...